### PR TITLE
one car

### PR DESCRIPTION
**Pull request by jkz**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -96,6 +96,7 @@ def parse_lines(lines):
 # Return solution score
 def score(cars, intersections, conf, destination):
     for t in range(conf.D):
+        print(t)
```
Comment (kevindhanna) [inline app.py]: don't like this print

File: app.py [hunk 2]
```diff
@@ -95,12 +95,16 @@ def parse_lines(lines):
 
 # Return solution score
 def score(cars, intersections, conf, destination):
+    pct = 0
     for t in range(conf.D):
-        print(t)
```
Comment (kevindhanna) [inline app.py]: yeah good

File: solutions/unused_streets.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+from collections import Counter
+from . import cycle
+
+def run(cars, streets, intersections, conf, destination):
+    # Remove unused streets
+    every = set(streets)
+    used = set([street.name for car in cars for street in car.route])
```
Comment (kevindhanna) [inline solutions/unused_streets.py:7]: this is fkd

**Review by kevindhanna — COMMENTED** [review]
❤️

**Review by Allcor — APPROVED** [review]

**Pull request merged**